### DISCUSSION TITLE
Support Vanish for Island Warp Message

### DISCUSSION THREAD

**Discussion by MythricHD** [BG-Software-LLC/SuperiorSkyblock2#434] [Denied Feature Requests] [answered]
When I am in vanish, the plugin supports certain aspects like not tabbing, etc but one that is missing is to hide when a vanished player warps to an island warp 
https://cdn.discordapp.com/attachments/873610744275566592/875062914501980210/unknown.png

I'm using premium vanish. I assume this was just forgotten when vanish support was added

**Comment by OmerBenGera** [comment, marked as answer]
Please create an issue, thats more like a bug than a new feature :)

**Reply by MythricHD** [reply]
Created https://github.com/BG-Software-LLC/SuperiorSkyblock2/issues/435, this can be closed. Thanks
